UserManagement.BL/UserManager.cs

```diff
@@ -31,6 +31,29 @@ public List<User> GetUsers()
             return repository.GetUsers().ToList();
         }
 
+        /// <summary>
+        /// Returns partial user information
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <returns></returns>
+        public PartialUser GetPartialUser(int userId)
+        {
+            var user = repository.GetUser(userId);
+
+            if (user != null)
+            {
+                var partialUser = new PartialUser()
+                {
+                    UserName = user.UserName,
+                    Email = user.Email,
+                };
+
+                return partialUser;
+            }
+
+            return null;
```

Comment:
Same as above. Return fast

Reply:
Or even better this could be done with a `constructor` that accepts `UserName` and `Email`  and make it a two line method. So this applies to upper most comment too.